src/Entity/ProductImage.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ProductRepository;
+use Doctrine\ORM\Mapping AS ORM;
+use DateTimeInterface;
+
+#[ORM\Entity(repositoryClass: ProductRepository::class)]
+#[ORM\Table(name: 'product_image')]
+class ProductImage
+{
+    #[ORM\Id, ORM\GeneratedValue(strategy: 'AUTO'), ORM\Column]
+    private int $id;
+
+    /**
+     * ProductImage constructor.
+     *
+     * @param string $title
+     * @param Product $product
+     * @param DateTimeInterface $createdAt
+     */
+    public function __construct(
+        #[ORM\Column]
+        private string $title,
+
+        #[ORM\OneToMany(mappedBy: Product::class)]
+        private Product $product,
+
+        #[ORM\Column]
+        private DateTimeInterface $createdAt = new \DateTimeImmutable('now'),
```

Comment:
it makes sense to store DateTime in UTC timezone, otherwise you may have errors on the webserver timezone change
```suggestion
        private DateTimeInterface $createdAt = new \DateTimeImmutable('now', new \DateTimeZone('UTC')),
```